scripts/remote_deployment/remote_run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Configure destination host
+USER=ubuntu
+HOST=10.46.49.242
```

Comment:
1. Wouldn't we need to proxy the wallet private keys going forward?
2. Let's remove these values and have them fed by the user, failing if not supplied? We can configure these as secrets on GitHub.

Reply:
I plan to remove this file.